changelog.txt

```diff
@@ -32,6 +32,7 @@
 * Fix - Update Cash App payments to avoid confirming on creation, resolving issues with generic payment failures in live mode.
 * Tweak - Add order lock for redirect payments.
 * Fix - Missing Stripe Fee and Stripe Payout details on orders that were captured manually.
+* Fix - Allow legacy `src_` payment methods linked to a saved credit card to be displayed on the checkout and My Account pages when the new checkout experience enabled.
```

Comment:
Nit: ...*is* enabled.

Reply:
Thanks for noticing this! 🙏 Fixed in [8da407f](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3513/commits/8da407fba754f34d64f34d0580f85baf4a742de6)